hveto/html.py

```diff
@@ -584,7 +678,10 @@ def write_summary(
 
     # scaffold plots
     if plots:
-        page.add(scaffold_plots(plots, nperrow=plotsperrow))
+        pparams = dict()
+        for i, p in enumerate(plots):
+            pparams[p] = {'title': HEADER_CAPTIONS[i]}
```

Comment:
I'm not sure I like this usage. Using a fixed mapping of `plots` to `HEAD_CAPTIONS` means that we have to remember what order the plots are given in, which seems fragile to me, especially since `scaffold_plots` is a generic function that just takes a list of plots.

I don't really have a better solution at this point, unfortunately.